examples/scripts/README.rst

```diff
@@ -238,6 +236,7 @@ PGM
    `sparsecode_poisson_pgm.py <sparsecode_poisson_pgm.py>`_
       Non-negative Poisson Loss Reconstruction (APGM)
    `denoise_tv_pgm.py <denoise_tv_pgm.py>`_
+      Isotropic Total Variation (Accelerated PGM)
```

Comment:
Not clear why this changed? Merge issue?

Reply:
Probably a merge issue. Removed.